packages/clay-panel/src/index.tsx

```diff
@@ -127,6 +127,7 @@ const ClayPanel: React.FunctionComponent<IProps> & {
 						)}
 						onClick={() => setInternalExpanded(!internalExpanded)}
 						role="tab"
+						type="button"
```

Comment:
@bryceosterhaus in that case, would we be able to use `ClayButton`?

Reply:
Good catch. I just updated to make it a little more robust and updated a few more components.